eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -115,6 +116,7 @@ def thread_process_iteration(client, manager, queue, block)
 
           #check that a message was received
           if response.messages.length > 0
+            tag_processing_thread
```

Comment:
should we move this to directly underneath the `Thread.new` line to ensure that all logging messages (errors, no message received etc include the `worker_id`) as currently this would only include it in logs after a message has been received (or is that the intention?)